devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -0,0 +1,220 @@
+/**
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *	Author: SRPOL
+ *	Date: 2019-02-18
+ */
+
+import groovy.json.JsonOutput
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition (name: "Zigbee Multi Button", namespace: "smartthings", author: "SmartThings") {
+		capability "Actuator"
+		capability "Battery"
+		capability "Button"
+		capability "Holdable Button"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Health Check"
+
+		fingerprint inClusters: "0000, 0001, 0003, 0007, 0020, 0B05", outClusters: "0003, 0006, 0019", manufacturer: "CentraLite", model:"3450-L", deviceJoinName: "Iris KeyFob 1"
```

Comment:
I just uploaded a new generic metadata using the new multi-component work to staging that should work for this device if you are on a staging environment as well.

Try adding a per-fingerprint vid pointing to `"generic-4-button"` and see if you can try it out in OneApp on a staging env.